sumologic/util_test.go

```diff
@@ -0,0 +1,202 @@
+package sumologic
```

Comment:
Unit test is great. Let's update some acceptance tests in `resource_sumologic_dashboard_test.go` to make sure we have coverage in acceptance tests too.

Reply:
I thought about it, but I didn't find a way to test that the plan is empty. Do you know how to write such a test?

Reply:
I can add a positive/negative variant (whichever lacks) there, but I don't see how it tests this PR, which only changes the emptiness of a plan.

Reply:
Yes to what Greg said. 

>I can add a positive/negative variant (whichever lacks) there, but I don't see how it tests this PR

With this PR, there should no diff in the acceptance test if we use one of the variants. Like `-24h` or `-1d` should not show any diff in acceptance test.

Reply:
Hi, I agree that there is a value in such tests. However, as I mentioned I didn't find a way to assert that a test is empty (only that it is not empty). Do you know how to do it/whether it is possible?

Reply:
>I didn't find a way to assert that a test is empty (only that it is not empty). 

You don't need to assert for anything. Having one of the time range variant like `-24h` in the acceptance test will ensure new code is tested. The backend will return `-1d` in such case, and the custom diff function will make sure there is no diff between state and the tf file.

Reply:
Thanks, I didn't know that the acc test already checks whether the plan was empty (maybe that's why I didn't find any flag to enforce such check :) ). I modified tests for dashboards, but I will also do it for monitors, as my change is applicable also there.